openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -96,27 +90,10 @@ def offer_html(self):
 
     @property
     def content_type_gating_enabled(self):
-        course_key = self.course_key
-        user = self.effective_user
-        is_enabled = None
-        course_masquerade = self.course_masquerade
-        if is_masquerading(user, course_key, course_masquerade):
```

Comment:
See mention elsewhere about calling this before the subsequent `is_masquerading_as_` checks later.